upstream.go

```diff
@@ -206,9 +206,28 @@ func (uc *upstreamConn) handleMessage(msg *irc.Message) error {
 	case "NOTICE":
 		uc.logger.Print(msg)
 
-		uc.forEachDownstream(func(dc *downstreamConn) {
-			dc.SendMessage(msg)
-		})
+		if msg.Prefix.User == "" && msg.Prefix.Host == "" {
+			uc.SendMessage(msg)
```

Comment:
This bounces the message back to the upstream server. We probably want to send it to downstream instead, or am I missing something?

Reply:
Right, thats a mistake